es-vue-base/src/lib-components/EsNavBarSearchBar.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="align-items-center w-100 d-flex justify-content-center position-relative">
+        <div class="d-flex align-items-center mx-auto justify-content-center w-100">
+            <es-form-input
+                id="searchBar"
+                class="mt-3 w-50"
+                v-bind="$attrs"
+                label-sr-only
+                :placeholder="label"
+                v-on="$listeners">
+                <template #prefixIcon>
+                    <icon-search />
+                </template>
+            </es-form-input>
+            <es-button class="ml-50">
```

Comment:
As of right now, this button can't do anything. I think the easiest way to wire things up from how you have it now is to emit an event from this button (say "searchButtonClicked"), which you can then listen to from the `nav` layout in the docs (or whatever downstream is implementing this). This listener will be responsible for navigating the user to the search results page, most likely passing query parameters which will allow the necessary API call to Algolia to be made. But all you have to worry about in the design system is verifying that the event works, so you could just do a `console.log` or something.